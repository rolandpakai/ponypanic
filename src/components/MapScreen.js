import { Fragment } from 'react';
import Header from './Header'; 
import MapContainer from './MapContainer'; 
import Footer from './Footer'; 

const MapScreen = () => {

  return (
    <Fragment> 
      <Header />  
      <MapContainer />
      <Footer />
    </Fragment>
  )
}

export default MapScreen;