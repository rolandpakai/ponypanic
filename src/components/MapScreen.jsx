import React from "react";

import Header from "./Header";
import MapContainer from "./MapContainer";
import Footer from "./Footer";

const MapScreen = () => {
  return (
    <>
      <Header />
      <MapContainer />
      <Footer />
    </>
  );
};

export default MapScreen;
