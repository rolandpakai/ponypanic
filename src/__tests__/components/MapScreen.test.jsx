import React from "react";
import renderer from "react-test-renderer";
import MapScreen from "../../components/MapScreen";

describe("MapScreen Component", () => {
  test("MapScreen renders", () => {
    const container = renderer.create(<MapScreen />).toJSON();

    expect(container).toMatchSnapshot();
  });
});
