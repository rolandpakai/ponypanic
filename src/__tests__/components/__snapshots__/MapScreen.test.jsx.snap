// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MapScreen Component MapScreen renders 1`] = `
Array [
  <div
    className="MuiBox-root css-0"
  >
    <div
      className="MuiContainer-root MuiContainer-maxWidthMd css-xdmu94-MuiContainer-root"
    >
      <div
        className="css-1ukijpk-MuiStack-root"
      >
        <img
          alt="title"
          className="header-img"
          src="./themes/pony/title.png"
          style={
            Object {
              "display": "block",
              "height": "110px",
              "objectFit": "contain",
            }
          }
        />
      </div>
    </div>
  </div>,
  <div
    className="MuiBox-root css-0"
  >
    <div
      className="MuiContainer-root MuiContainer-maxWidthMd css-xdmu94-MuiContainer-root"
    >
      <div
        className="css-1av9md8-MuiStack-root"
      >
        <div
          className="MuiBox-root css-1as3uml"
        >
          <div
            className="MuiBox-root css-gaqaxa"
            id="id-undefined"
          />
        </div>
      </div>
    </div>
    <div
      className="MuiContainer-root MuiContainer-maxWidthXs css-nzinn5-MuiContainer-root"
    >
      <div
        className="css-vankme-MuiStack-root"
      >
        <button
          className="MuiButtonBase-root MuiButton-root MuiLoadingButton-root MuiButton-contained MuiButton-containedError MuiButton-sizeLarge MuiButton-containedSizeLarge MuiButton-fullWidth MuiButton-root MuiLoadingButton-root MuiButton-contained MuiButton-containedError MuiButton-sizeLarge MuiButton-containedSizeLarge MuiButton-fullWidth css-dat04h-MuiButtonBase-root-MuiButton-root-MuiLoadingButton-root"
          disabled={false}
          id=":r0:"
          onBlur={[Function]}
          onClick={[Function]}
          onContextMenu={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          HERO TURN
        </button>
      </div>
    </div>
  </div>,
  <div
    className="MuiBox-root css-5mtktj"
  >
    <div
      className="MuiBox-root css-1g360fb"
    >
      <div
        className="css-w4z10b-MuiStack-root"
      >
        <a
          className="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-1cghntr-MuiTypography-root-MuiLink-root"
          href="https://www.linkedin.com/in/roland-p%C3%A1kai-6a2041249/"
          onBlur={[Function]}
          onFocus={[Function]}
          target="_blank"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root MuiSvgIcon-fontSizeLarge css-tzssek-MuiSvgIcon-root"
            data-testid="LinkedInIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M19 3a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h14m-.5 15.5v-5.3a3.26 3.26 0 0 0-3.26-3.26c-.85 0-1.84.52-2.32 1.3v-1.11h-2.79v8.37h2.79v-4.93c0-.77.62-1.4 1.39-1.4a1.4 1.4 0 0 1 1.4 1.4v4.93h2.79M6.88 8.56a1.68 1.68 0 0 0 1.68-1.68c0-.93-.75-1.69-1.68-1.69a1.69 1.69 0 0 0-1.69 1.69c0 .93.76 1.68 1.69 1.68m1.39 9.94v-8.37H5.5v8.37h2.77z"
            />
          </svg>
        </a>
        <a
          className="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-1cghntr-MuiTypography-root-MuiLink-root"
          href="https://github.com/rolandpakai/ponypanic"
          onBlur={[Function]}
          onFocus={[Function]}
          target="_blank"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root MuiSvgIcon-fontSizeLarge css-tzssek-MuiSvgIcon-root"
            data-testid="GitHubIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M12 1.27a11 11 0 00-3.48 21.46c.55.09.73-.28.73-.55v-1.84c-3.03.64-3.67-1.46-3.67-1.46-.55-1.29-1.28-1.65-1.28-1.65-.92-.65.1-.65.1-.65 1.1 0 1.73 1.1 1.73 1.1.92 1.65 2.57 1.2 3.21.92a2 2 0 01.64-1.47c-2.47-.27-5.04-1.19-5.04-5.5 0-1.1.46-2.1 1.2-2.84a3.76 3.76 0 010-2.93s.91-.28 3.11 1.1c1.8-.49 3.7-.49 5.5 0 2.1-1.38 3.02-1.1 3.02-1.1a3.76 3.76 0 010 2.93c.83.74 1.2 1.74 1.2 2.94 0 4.21-2.57 5.13-5.04 5.4.45.37.82.92.82 2.02v3.03c0 .27.1.64.73.55A11 11 0 0012 1.27"
            />
          </svg>
        </a>
      </div>
    </div>
    <div
      className="MuiBox-root css-1g360fb"
    >
      <div
        className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary  css-1yk1gt9-MuiInputBase-root-MuiOutlinedInput-root-MuiSelect-root"
        onClick={[Function]}
      >
        <div
          aria-expanded="false"
          aria-haspopup="listbox"
          className="MuiSelect-select MuiSelect-outlined MuiInputBase-input MuiOutlinedInput-input css-11u53oe-MuiSelect-select-MuiInputBase-input-MuiOutlinedInput-input"
          onBlur={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onMouseDown={[Function]}
          role="button"
          tabIndex={0}
        >
          Pony
        </div>
        <input
          aria-hidden={true}
          className="MuiSelect-nativeInput css-yf8vq0-MuiSelect-nativeInput"
          onAnimationStart={[Function]}
          onChange={[Function]}
          tabIndex={-1}
          value="pony"
        />
        <svg
          aria-hidden={true}
          className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiSelect-icon MuiSelect-iconOutlined css-hfutr2-MuiSvgIcon-root-MuiSelect-icon"
          data-testid="ArrowDropDownIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M7 10l5 5 5-5z"
          />
        </svg>
        <fieldset
          aria-hidden={true}
          className="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
        >
          <legend
            className="css-ihdtdm"
          >
            <span
              className="notranslate"
            >
              â€‹
            </span>
          </legend>
        </fieldset>
      </div>
    </div>
  </div>,
]
`;
